jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/EnvUtil.java

```diff
@@ -126,26 +127,22 @@ public static boolean greaterOrEqualsVersion(String versionA, String versionB) {
      */
     public static List<String[]> splitOnLastColon(List<String> listToSplit) {
         if (listToSplit != null) {
-            return Lists.transform(listToSplit, SPLIT_ON_LAST_COLON);
+            return listToSplit.stream().map(SPLIT_ON_LAST_COLON).collect(Collectors.toList());
         }
         return Collections.emptyList();
     }
 
-    private static final Function<String, Iterable<String>> COMMA_SPLITTER = new Function<String, Iterable<String>>() {
-        private Splitter COMMA_SPLIT = Splitter.on(",").trimResults().omitEmptyStrings();
-
+    private static final Function<String, List<String>> COMMA_SPLITTER = new Function<String, List<String>>() {
         @Override
-        public Iterable<String> apply(String input) {
-            return COMMA_SPLIT.split(input);
+        public List<String> apply(String input) {
+            return Arrays.stream(input.split(","))
+                    .map(TRIM)
+                    .filter(NOT_EMPTY)
+                    .collect(Collectors.toList());
         }
     };
 
-    private static final Predicate<String> NOT_EMPTY = new Predicate<String>() {
-        @Override
-        public boolean apply(@Nullable String s) {
-            return s != null && !s.isEmpty();
-        }
-    };
+    private static final Predicate<String> NOT_EMPTY = s -> s != null && !s.isEmpty();
 
     private static final Function<String, String> TRIM = new Function<String, String>() {
```

Comment:
Try using the more specialised Functional Interface `UnaryOperator<String>`

Reply:
Done